### PR TITLE
Data Explanation Guide

### PR DESCRIPTION
**Pull request by lkiesow**
This marks the beginning of collaborative work on a data guide.

#### Things to explain

- [x] Show that we usually only have measurements from a part of the day:
  https://demo.emissions-api.org/api/v1/average.json/?country=CA&begin=2019-02-01&end=2019-03-01
    - [ ] Build graph visualizing the measurement intervals
- [x] Measurement durations for a country depend on the size of the country
- [x] Measurements may depend of the time of day (e.g. rush hour)
- [x] Measurements are not ground level values

#### Additional todo:

- [ ] Convert single example into a guides section

### CODE REVIEW

**Comment by tibroc** [comment]
While I did not visualize the checked boxes above they are taken care of with explanations in the text. This is a draft that can be extended with more graphics in the future.

I found it difficult to write less text without not being too superficial. However, it already is quite a long read and could probably use some more figures (which in turn will make it even longer).

File: guides/data.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+For the second factor, it is important to keep in mind that when we talk about the data we are not talking about ground measurements, but about everything that occurs in the atmosphere.
+
+Another thing that you will notice when working with the data is that there are sometimes what appear to be holes in the data:
+
+![cloudy](/assets/img/cloudy.png)
+
+This is due to a reason that might seem somewhat obvious:
+you need to see the ground to be able to measure the concentrations.
+In other words, where there are clouds, there are no measurements.
```
Comment (EmbeddedHenni) [inline guides/data.md]:
```suggestion
In other words, when there are clouds, there are no measurements.
```

File: guides/data.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+You can see all the data points at the time they were being recorded.
+The whole scan takes about a minute and a half in realtime to pass over the whole area of Germany.
+But the total time to complete a scan depends of course on the size of the region you are interested in.
+If your region of interest spans more than 2600 km in width (like Canada or Russia), i.e. it needs more than one orbit to scan, it will be about two hours longer per additional orbit.
+This is because the satellite needs approximately one hour to scan one side of the earth and then another hour to pass the other side (where it is night) and come back up.
+You can get an impression of this from the following animation that iterates through the data of all the orbits of one day, one after another:
+
+![polar night](/assets/img/world_201902_01-02.gif)
+
+For analyses of air quality data this means that you can only see temporal patterns that manifest themselves at scales bigger than one day.
```
Comment (EmbeddedHenni) [inline guides/data.md]:
```suggestion
For analyses of air quality data this means that you can only see temporal patterns that manifest themselves at scales longer than one day.
```

**Review by EmbeddedHenni — APPROVED** [review]
lgtm

File: data.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+You can see all the data points at the time they were being recorded.
+The whole scan takes about a minute and a half in realtime to pass over the whole area of Germany.
+But the total time to complete a scan depends of course on the size of the region you are interested in.
+If your region of interest spans more than 2600 km in width (like Canada or Russia), i.e. it needs more than one orbit to scan, it will be about two hours longer per additional orbit.
+This is because the satellite needs approximately one hour to scan one side of the earth and then another hour to pass the other side (where it is night) and come back up.
+You can get an impression of this from the following animation that iterates through the data of all the orbits of one day, one after another:
+
+![polar night](/assets/img/world_201902_01-02.gif)
+
+For analyses of air quality data this means that you can only see temporal patterns that manifest themselves at scales bigger than one day.
```
Comment (EmbeddedHenni) [inline data.md]:
```suggestion
For analyses of air quality data this means that you can only see temporal patterns that manifest themselves at scales longer than one day.
```

**Review by EmbeddedHenni — APPROVED** [review]

File: data.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+# Sentinel-5P Data Guide
+
+When we think about satellite data we tend to think of the [heatmap-like overlays](https://www.esa.int/Applications/Observing_the_Earth/Copernicus/Sentinel-5P/Nitrogen_dioxide_pollution_mapped) of some variable over a map.
+These images usually suggest that we have quite a dense data set that gives us a complete and unambiguous impression of the world.
+In practice these maps are usually made by aggregating a lot of data either over a longer time period.
+While it is pretty amazing that you can measure things on earth from outer space in such a high resolution at such a huge scale,
+when you look at the raw data you quickly realize that they also come with some important limitations.
+This is why in this text we would like to give you a short overview about what you can expect from this data and what kind of questions you can tackle with it.
+
+## What you (can) see is what you get
```
Comment (tibroc) [inline data.md]:
```suggestion
## What you see is what you get
```

File: data.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+This is due to a reason that might seem somewhat obvious:
+you need to see the ground to be able to measure the concentrations.
+In other words, where there are clouds, there are no measurements.
+Additionally, for Emissions API we discard all the points that are below the quality level recommended by the ESA and thus you will see no data for which the estimate is doubtful.
+
+## Spatial Resolution
+
+Since the satellite scans lines of pixels while orbiting around the earth, the spatial resolution is determined by two factors:
+the number of pixels in the direction of the scan line and the recording frequency in the direction of flight
+(see figure 48 in the [mission page](https://directory.eoportal.org/web/eoportal/satellite-missions/c-missions/copernicus-sentinel-5p).
```
Comment (tibroc) [inline data.md]:
```suggestion
(see figure 48 on the [mission page](https://directory.eoportal.org/web/eoportal/satellite-missions/c-missions/copernicus-sentinel-5p)).
```

**Pull request merged**